Crash when tapping back on draft preview

Test Online-WPCom-4

### Expected behavior
When I open a preview and tap back the app doesn't crash.

```
2019-10-01 10:38:01.954 8689-8689/org.wordpress.android E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.android, PID: 8689
    java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.posts.EditPostActivity}: java.lang.IllegalArgumentException: PostLoadingState wrong value 6
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3268)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3407)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7343)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:933)
     Caused by: java.lang.IllegalArgumentException: PostLoadingState wrong value 6
        at org.wordpress.android.ui.posts.EditPostActivity$PostLoadingState.fromInt(EditPostActivity.java:343)
        at org.wordpress.android.ui.posts.EditPostActivity.onCreate(EditPostActivity.java:529)
        at android.app.Activity.performCreate(Activity.java:7802)
        at android.app.Activity.performCreate(Activity.java:7791)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1299)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java
```

### Steps to reproduce the behavior
- Probably crashes only with the "Don't keep activities" option
1. On the post list, Draft tab selected
2. On a Draft without local changes, Tap “… More” -> “View”
3. Remote view (webview) should open
4. Tap back
5. The app crashes

--- a/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java
@@ -339,11 +339,7 @@ public class EditPostActivity extends AppCompatActivity implements
         }
 
         public static PostLoadingState fromInt(int value) {
-            if (value < 0 || value >= values().length) {
-                throw new IllegalArgumentException("PostLoadingState wrong value " + value);
-            }
-
-            PostLoadingState state = NONE;
+            PostLoadingState state = null;
 
             for (PostLoadingState item : values()) {
                 if (item.mValue == value) {
@@ -352,6 +348,9 @@ public class EditPostActivity extends AppCompatActivity implements
                 }
             }
 
+            if (state == null) {
+                throw new IllegalArgumentException("PostLoadingState wrong value " + value);
+            }
             return state;
         }
     }

